301-sonarqube-azuresql/azuredeploy.json

```diff
@@ -0,0 +1,399 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sqVM_AppName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Name of the VM that SonarQube will be installed upon"
+      }
+    },
+    "sq_PublicIP_DnsPrefix": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 32,
+      "metadata": {
+        "description": "The prefix of the public URL for the VM on the Internet. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "sqVM_AppAdmin_UserName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Admin account name for the SonarQube VM"
+      }
+    },
+    "sqVM_AppAdmin_Password": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the SonarQube VM Admin account"
+      }
+    },
+
+
+    "sqDB_Admin_UserName": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Admin account name for Azure SQL Server"
+      }
+    },
+    "sqDB_Admin_Password": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for Azure SQL Server Admin account"
+      }
+    },
+    "sqDB_ServerName": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 10,
+      "defaultValue": "sonarsql",
+      "metadata": {
+        "description": "Name of Azure SQL Server (limit to 10 chars or less)"
+      }
+    },
+    "sqDB_DBName": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "sonar",
+      "metadata": {
+        "description": "Name of the SonarQube DB on the Azure SQL Server"
+      }
+    },
+    "sqDB_DBEdition": {
+      "type": "string",
+      "defaultValue": "Basic",
+      "allowedValues": [
+        "Basic",
+        "Business",
+        "Premium",
+        "Standard",
+        "Web"
+      ],
+      "metadata": {
+        "description": "Edition of Azure SQL Server to create"
+      }
+    },
+    "sqStorage_AcctType": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ],
+      "metadata": {
+        "description": "Type of Azure Storage Acct to create"
+      }
+    }
+  },
+  "variables": {
+    "GitHubRoot": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master",
```

Comment:
this still needs to be resolved; it should be a parameter called _artifactsLocation instead of a variable